.gitignore

```diff
@@ -1 +1,4 @@
-.venv
\ No newline at end of file
+.venv
+*.pyc
```

Comment:
if you want to untrack all these pycache files so Git ignores them, [check this out.](https://gist.github.com/mcrd25/1b2718054b65f4720fa582c6a4032217)

Reply:
Cool!